apps/sv/frontend/src/routes/createProposal.tsx

```diff
@@ -5,13 +5,24 @@ import { useSearchParams } from 'react-router-dom';
 import { createProposalActions } from '../utils/governance';
 import { SelectAction } from '../components/forms/SelectAction';
 import { CreateProposalForm } from '../components/forms/CreateProposalform';
+import { SupportedActionTag } from '../utils/types';
+import { UpdateSvRewardWeightForm } from '../components/forms/UpdateSvRewardWeightForm';
 
 export const CreateProposal: React.FC = () => {
   const [searchParams, _] = useSearchParams();
   const action = searchParams.get('action');
   const selectedAction = createProposalActions.find(a => a.value === action);
+
+  const onSubmit = () => Promise.resolve();
+
   if (selectedAction) {
-    return <CreateProposalForm action={selectedAction} />;
+    const a = selectedAction.value as SupportedActionTag;
+    switch (a) {
```

Comment:
This is where we eventually enumerate all the forms and render them based on action selected.